templates/Simple/invoice.php

```diff
@@ -106,21 +106,21 @@
 	</thead>
 	<tbody>
 		<?php foreach ( $this->get_order_items() as $item_id => $item ) : ?>
-			<tr class="<?php echo apply_filters( 'wpo_wcpdf_item_row_class', 'item-'.$item_id, $this->get_type(), $this->order, $item_id ); ?>">
+			<tr class="<?php echo apply_filters( 'wpo_wcpdf_item_row_class', 'item-'.$item_id, esc_attr( $this->get_type() ), $this->order, $item_id ); ?>">
 				<td class="product">
 					<?php $description_label = __( 'Description', 'woocommerce-pdf-invoices-packing-slips' ); // registering alternate label translation ?>
-					<span class="item-name"><?php echo $item['name']; ?></span>
+					<span class="item-name"><?php echo esc_html( $item['name'] ); ?></span>
 					<?php do_action( 'wpo_wcpdf_before_item_meta', $this->get_type(), $item, $this->order  ); ?>
-					<span class="item-meta"><?php echo $item['meta']; ?></span>
+					<span class="item-meta"><?php echo esc_html( $item['meta'] ); ?></span>
```

Comment:
Should be `wp_kses_post()`.  Item meta HTML is now escaped.

<img width="659" alt="Schermafbeelding 2022-10-20 om 10 34 37" src="https://user-images.githubusercontent.com/19350150/196899189-dcd7f190-1b5e-418e-a53a-7355bd6e6224.png">